api/run_eval.py

```
@@ -137,9 +137,14 @@ def transcribe_with_retry(
                             ):
                                 errors = status["job"]["errors"]
-                                if "message" in errors[-1] and "failed to fetch file" in errors[-1]["message"]:
+                                if (
+                                    "message" in errors[-1]
+                                    and "failed to fetch file" in errors[-1]["message"]
+                                ):
                                     retries = max_retries + 1
-                                    raise Exception(f"could not fetch URL {audio_url}, not retrying")
+                                    raise Exception(
+                                        f"could not fetch URL {audio_url}, not retrying"
+                                    )
```

Ruff (0.13.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)